### PR TITLE
Closes #2332: Fix turboButton focusLeft

### LINKED ISSUE #2332: Focus handling for turbo mode button
## Steps to reproduce
- open fftv
- navigate to turbo button
- press left on DPAD

### Expected behavior
- nothing

### Actual behavior
- pocket or pinned_tile gain focus

### Device information
* Fire TV device: ?
* Latest affected Firefox version: ?
* Earliest affected Firefox version: ?

### PR DESCRIPTION
**Pull request by psymoon**
### Pull Request checklist

- [x] This PR includes thorough **tests** or an explanation of why it does not
- [x] This PR includes a **CHANGELOG entry** or does not need one
- [x] I have considered adding **QA labels** on the associated issue (not this PR; `qa-ready` or `qa-denied`)

### CODE REVIEW

File: app/src/main/java/org/mozilla/tv/firefox/focus/FocusRepo.kt [hunk 1]
```diff
@@ -241,6 +244,25 @@ class FocusRepo(
...
+        sessionState: SessionRepo.State
+    ): State {
+
+        assert(focusedTurboButtonNode.viewId == R.id.turboButton)
+
+        val nextFocusLeftId = when {
+            // TODO: this is a duplicating existing logic in the ToolbarVM, may fall out of sync
+            sessionState.currentUrl == URLs.APP_URL_HOME -> R.id.turboButton
+            else -> R.id.pinButton
+        }
```
Comment (Baron-Severin) [inline app/src/main/java/org/mozilla/tv/firefox/focus/FocusRepo.kt]: There's an edge case here where pin is disabled and forward is enabled.

-> Open app
-> Select any tile
-> Hit back button

Comment (psymoon) [reply app/src/main/java/org/mozilla/tv/firefox/focus/FocusRepo.kt]: Ah, nice one

Comment (psymoon) [reply app/src/main/java/org/mozilla/tv/firefox/focus/FocusRepo.kt]: @Baron-Severin hmm it looks like we can't go back to `about:home` after selecting a tile and according to `ToolbarViewModel`, `pinEnabled = !sessionState.currentUrl.isEqualToHomepage()`

Comment (Baron-Severin) [reply app/src/main/java/org/mozilla/tv/firefox/focus/FocusRepo.kt]: I'm not sure of the code path, but I'm able to reproduce the state by following those steps.

Screen Shot 2019-05-24 at 1 06 51 PM

Comment (psymoon) [reply app/src/main/java/org/mozilla/tv/firefox/focus/FocusRepo.kt]: Ah okay so you can go back from within WebRenderFragment but not in Overlay 🤔

**Pull request merged**